### ISSUE TITLE
Getting babelHelpers is not defined (v3.0.6)

### ISSUE THREAD

**Issue by donysukardi** [rollup/rollup-plugin-babel#228]
Changes in https://github.com/rollup/rollup-plugin-babel/commit/f8182c607b384d1c014b0e7cb70d4feb69073003 (v3.0.6) seem to be breaking my build which was previously working fine.

I am getting `babelHelpers is not defined` error.

![image](https://user-images.githubusercontent.com/410792/42136328-b2c2d31e-7d8b-11e8-9998-f9faced79a18.png)

```javascript
//.babelrc
{
  ...,
  plugins: ["babel-plugin-external-helpers", ...]
}
```

It seems to be working fine after I remove `"babel-plugin-external-helpers"`. Should I not include that plugin?

**Comment by Andarist** [comment]
You should include it when using 3.x version. Could you share ur repository? Or some other with the issue reproduced?

**Comment by sonaye** [comment]
Same here, had to downgrade to 3.0.5 to get it working.

<img width="1212" alt="screen shot 2018-07-01 at 10 43 07 pm" src="https://user-images.githubusercontent.com/23000873/42138078-53ea7730-7d80-11e8-8a27-66c323523aac.png">

To reproduce `yarn build` https://github.com/sonaye/react-is-responsive with (3.0.5 and 3.0.6) and compare `lib/index.js`.

**Comment by Andarist** [comment]
I've reverted the last change and published a new version - bascailly 3.0.7 is the same as 3.0.5 now. Need to figure out how to fix the issue so the change from 3.0.6 can get published too.

**Comment by Andarist** [comment]
Closing this as solved, not sure if what got merged in and released as 3.0.6 is possible. I've proposed [alternative solution](https://github.com/rollup/rollup-plugin-babel/pull/227#issuecomment-401634742) for a person who made a PR.
